components/elm/bld/CLMBuildNamelist.pm

```diff
@@ -3402,17 +3402,17 @@ sub write_output_files {
   # CLM component
   my @groups;
   if ( $physv->as_long() == $physv->as_long("clm4_0") ) {
-    @groups = qw(clm_inparm);
+    @groups = qw(elm_inparm);
     # Eventually only list namelists that are actually used when CN on
     #if ( $nl_flags->{'bgc_mode'}  eq "cn" ) {
       push @groups, "ndepdyn_nml";
       push @groups, "pdepdyn_nml";
     #}
   } else {
-    @groups = qw(clm_inparm ndepdyn_nml pdepdyn_nml popd_streams light_streams lai_streams clm_canopyhydrology_inparm 
-                 clm_soilhydrology_inparm dynamic_subgrid finidat_consistency_checks dynpft_consistency_checks 
-                 clmu_inparm clm_soilstate_inparm clm_pflotran_inparm betr_inparm);
-    #@groups = qw(clm_inparm clm_canopyhydrology_inparm clm_soilhydrology_inparm 
+    @groups = qw(elm_inparm ndepdyn_nml pdepdyn_nml popd_streams light_streams lai_streams elm_canopyhydrology_inparm 
+                 elm_soilhydrology_inparm dynamic_subgrid finidat_consistency_checks dynpft_consistency_checks 
+                 elmu_inparm elm_soilstate_inparm elm_pflotran_inparm betr_inparm);
+    #@groups = qw(elm_inparm clm_canopyhydrology_inparm clm_soilhydrology_inparm 
```

Comment:
These two namelist sets should also be renamed

Reply:
A recent commit fixes it.